src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -11,6 +11,7 @@
     <RazorCompileOnBuild>true</RazorCompileOnBuild>
     <ResolvedRazorCompileToolset>RazorSdk</ResolvedRazorCompileToolset>
     <PrecompiledAssemblyTitle Condition=" '$(PrecompiledAssemblyTitle)' == '' ">$(RazorTargetName).dll</PrecompiledAssemblyTitle>
+    <IncludeContentInPack>false</IncludeContentInPack>
```

Comment:
What's this about? I don't have any context here

Reply:
We're picking up a bug fix made to the Identity UI package.